CapstoneProject/Assets/Scripts/Core/Main Menu/MainMenu.cs

```diff
@@ -29,13 +29,31 @@ private void Start()
 
     public void Continue()
     {
+
+        PlayerSaveData data = SaveSystem.LoadPlayer();
+        if (data == null)
+        {
+            Debug.Log("No save data found!");
+            //TODO: Add UI for this!
+        }
+        else if (data!= null)
+        {
+            Debug.Log(data.playerScene);
+            SceneManager.LoadScene(data.playerScene);
+        }
+        else
+        {
+            Debug.Log("Data isn't null but it also isn't not null. May God have mercy on our souls.");
+            //TODO: delete OS if this happens
```

Comment:
ummm....let's not and say we never did. why do we even have an else if and not just an else?